server/database/queries/getForms.js

```diff
@@ -0,0 +1,15 @@
+const db_connection = require('../config/db_connection');
+
+const getForms = (employeeID) => {
+	const sql = {
+		text:
+			' SELECT e.name employee_form,f.reviewer_name ,q.context question,eq.result_rate rate_answer,eq.answer context_answer,q.role FROM questions q, employees e,employees_questions eq,form_questions fq,forms f WHERE e.id=eq.employee_id AND q.id=eq.question_id AND fq.form_id=f.id AND fq.question_id=q.id AND eq.employee_id=$1;',
+		values: [employeeID],
+	};
+
+	return db_connection
+		.query(sql)
+		.then((res) => res.rows)
+		.catch((err) => err);
```

Comment:
How will we handle this error?
Has the team discussed a process for handling errors?

Reply:
We will handle errors by middlewares, we will accept consol.logs now and the next step will be the middleware!